pkg/apis/openebs.io/v1alpha1/cstorvolume.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+message VolumeSnapRequest {
+	string command = 1;
+	string volume = 2;
+	string snapname = 3;
+}
+
+message VolumeSnapResponse {
+	bytes status = 1;
+}
+
+service RunSnapCommand {
```

Comment:
We should create snapshot, clone, etc methods here right? Then also we dont need to command string ..